indico/htdocs/sass/partials/_core.scss

```diff
@@ -64,7 +64,7 @@
 }
 
 .weak-hidden {
-    display: none;
+    display: none !important;
```

Comment:
if you don't need to change it from JS simply use `hidden`, otherwise inline CSS